Editor/UI/Framework/SharedStyles.cs

```diff
@@ -128,6 +131,35 @@ public static GUIStyle TextAreaWithDynamicSize
             }
         }
 
+        public static GUIStyle WelcomeLabel
```

Comment:
"WelcomeLabel" sounds specific to the `ProjectAuditorWindow`. Can we generalize it by naming it "TitleLabel" ?

Reply:
> "WelcomeLabel" sounds specific to the `ProjectAuditorWindow`. Can we generalize it by naming it "TitleLabel" ?

Yes, makes sense.